core/api/src/main/java/com/blazebit/persistence/spi/DbmsDialect.java

```diff
@@ -205,6 +206,22 @@
     public boolean supportsJoinsInRecursiveCte();
 
     /**
+     * Returns true if the dbms supports row value constructor syntax, false otherwise.
+     *
+     * @return Whether row value constructor syntax is supported by the dbms
+     * @since 1.2.0
+     */
+    public boolean supportsRowValueConstructor();
+
+    /**
+     * Returns true if the dbms supports <, <=, >, >=, =, <> comparison operations for row values.
```

Comment:
How is this misleading? The description states that this property is true if all of the presented operators are supported. Cannot be expressed clearer IMHO. The description implies that the sole support of <>, = operators leads to the property being false.
Since the support for row value constructor is independent from the support for row value comparison, I figured that it makes sense to introduce separate properties. But you are right, the `supportsFullRowValueComparison` would suffice at the moment.